### DISCUSSION TITLE
What piece of technology did IBM develop in 1973 that they nicknamed Winchester?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#343] [Galaxy Trivia] [answered]
**Hint:** 📝

**Comment by GameExplorer** [comment]
IBM 3340

**Comment by thuongtruong1009** [comment, marked as answer]
The piece of technology that IBM developed in 1973 and nicknamed "Winchester" was the first hard disk drive (HDD) with two read/write heads that could access both sides of a disk platter, increasing storage capacity and access speed. The IBM 3340, as it was officially called, had a storage capacity of 30 megabytes and was used in IBM's System/370 mainframe computers. The name "Winchester" was chosen because it was the code name for the project and also the name of a popular 1970s-era rifle.

**Comment by peterbaccaro** [comment]
In 1973, IBM developed a type of hard disk drive that they nicknamed "Winchester". The IBM 3340, also known as the Winchester, was one of the first commercially available hard disk drives, and it helped to establish the dominance of the hard disk as the primary storage medium for computers.
